doc/sam.adoc

```diff
@@ -25,9 +25,11 @@ include::tutorial.adoc[]
 include::reference/barrier.adoc[]
 include::reference/condition_variable.adoc[]
 include::reference/mutex.adoc[]
+include::reference/shared_mutex.adoc[]
```

Comment:
In `config.adoc`: "The user needs can include `boost/sam/src.hpp` as an alternative to linking." syntactically incorrect